### PR TITLE
Update auto_instrumentation.md

### PR DESCRIPTION
**Pull request by yahya-mouman**
Update docs to include all the newly traced methods for langchain.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/llm_observability/setup/auto_instrumentation.md [hunk 1]
```diff
@@ -72,15 +72,23 @@ The LangChain integration instruments the following methods:
...
 - [Embeddings][17]
   - OpenAI : `OpenAIEmbeddings.embed_documents()`, `OpenAIEmbeddings.embed_query()`
+- [Tools][21]
+  - `BaseTool.invoke()`, `BaseTool.ainvoke()`
+- [Retrieval][22]
+  - langchain_community.<vectorstores>, langchain_pinecone, langchain_qdrant, langchain_milvus
+  - `similarity_search()`, `similarity_search_with_score()`, `similarity_search_by_vector_with_score()`, `similarity_search_with_score_by_vector()`, `similarity_search_by_vector()`
 
-**Note:** The LangChain integration does not yet support tracing streamed calls.
+**Note:** The LangChain integration only supports tracing streamed calls from llm, chat_model and chain.
```
[the remark this answers is not in the document]
Comment (yahya-mouman) [reply content/en/llm_observability/setup/auto_instrumentation.md]: Done !

**Comment by estherk15** [comment]
Whenever this is ready to merge, you can add it to the merge queue by commenting `/merge`

**Review by sabrenner — APPROVED** [review]

File: content/en/llm_observability/setup/auto_instrumentation.md [hunk 2]
```diff
@@ -72,15 +72,21 @@ The LangChain integration instruments the following methods:
...
+  - `chain.stream()`, `chain.astream()`
 - [Embeddings][17]
   - OpenAI : `OpenAIEmbeddings.embed_documents()`, `OpenAIEmbeddings.embed_query()`
-
-**Note:** The LangChain integration does not yet support tracing streamed calls.
+- [Tools][21]
+  - `BaseTool.invoke()`, `BaseTool.ainvoke()`
+- [Retrieval][22]
+  - langchain_community.<vectorstores>, langchain_pinecone, langchain_qdrant, langchain_milvus
+  - `similarity_search()`, `similarity_search_with_score()`, `similarity_search_by_vector_with_score()`, `similarity_search_with_score_by_vector()`, `similarity_search_by_vector()`
```
Comment (yahya-mouman) [inline content/en/llm_observability/setup/auto_instrumentation.md:89]:
```suggestion
- [Retrieval][22]
  - langchain_community.<vectorstores>, langchain_pinecone
  - `similarity_search()`
```

Comment (yahya-mouman) [reply content/en/llm_observability/setup/auto_instrumentation.md:89]: Removing the rest and differing it to a later PR to unblock this one.

**Pull request merged**